### ISSUE TITLE
Error in Terminal

### ISSUE THREAD

**Issue by Capreyon** [microsoft/vscode#79745]
Issue Type: <b>Bug</b>

I won't able to type anything in the Terminal.

VS Code version: Code 1.37.1 (f06011ac164ae4dc8e753a3fe7f9549844d15e35, 2019-08-15T16:17:55.855Z)
OS version: Windows_NT x64 6.2.9200

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-8300H CPU @ 2.30GHz (8 x 2304)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_deferred_display_list: disabled_off<br>skia_renderer: disabled_off<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>viz_display_compositor: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|7.86GB (1.95GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (4)</summary>

Extension|Author (truncated)|Version
---|---|---
VBScript|lug|0.0.2
mssql|ms-|1.6.0
python|ms-|2019.8.30787
cpptools|ms-|0.25.0

</details>

**Comment by RMacfarlane** [comment]
@Capreyon Can you describe the problem a little bit more? When does this happen? Do you see an error message?

**Comment by Capreyon** [comment]
@RMacfarlane Yeah Sure, Actually the problem is same as some previous posts like I won't be able to type anything in the Terminal of the VS code, even you won't be able to see the current directory path in the Terminal as it is visible in normal working mode like (Sudhanshu@LAPTOP-JVKE5B8Q MINGW64 ~/Desktop/AI_lab) then I tried to uninstall and again install VS code but still it won't work 
then the last solution I tried to reinstall but this time I changed the drive location while installation and It worked.
